MovieAppPetProject/MovieAppPetProject/Login/LoginViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LoginViewController.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/04/15.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController, LoginViewModelDelegate {
+
+    // MARK: - Outlets
+    @IBOutlet private weak var usernameInput: UITextField!
+    @IBOutlet private weak var passwordInput: UITextField!
+
+    // MARK: - Variable
+    private var loginViewModel: LoginViewModel!
```

Comment:
force unwrapping?
This shouldn't be allowed with swiftlint?
Please instantiate this viewModel properly
private lazy var loginViewModel = LoginViewModel(delegate: self)
Please remove it from viewDidLoad.

Reply:
changes made in login cleanup PR